From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Bradley <daniel@pulumi.com>
Date: Wed, 22 May 2024 13:26:25 +0100
Subject: [PATCH] Add combined cosmosdb cassandra cluster test

Unclear why this patch is applied. Likely added in error due to a merge.

diff --git a/internal/services/cosmos/cosmosdb_cassandra_cluster_resource_test.go b/internal/services/cosmos/cosmosdb_cassandra_cluster_resource_test.go
index e6a2b93e01..53efd0fc1f 100644
--- a/internal/services/cosmos/cosmosdb_cassandra_cluster_resource_test.go
+++ b/internal/services/cosmos/cosmosdb_cassandra_cluster_resource_test.go
@@ -18,6 +18,18 @@ import (
 
 type CassandraClusterResource struct{}
 
+func TestAccCassandraCluster(t *testing.T) {
+	// NOTE: this is a combined test rather than separate split out tests due to all below TCs sharing same sp
+	acceptance.RunTestsInSequence(t, map[string]map[string]func(t *testing.T){
+		"basic": {
+			"basic":          testAccCassandraCluster_basic,
+			"requiresImport": testAccCassandraCluster_requiresImport,
+			"complete":       testAccCassandraCluster_complete,
+			"update":         testAccCassandraCluster_update,
+		},
+	})
+}
+
 func testAccCassandraCluster_basic(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_cosmosdb_cassandra_cluster", "test")
 	r := CassandraClusterResource{}
